akka-docs/rst/scala/http/client-side/connection-level.rst

```diff
@@ -55,6 +55,9 @@ The connection can also be closed by the server.
 An application can actively trigger the closing of the connection by completing the request stream. In this case the
 underlying TCP connection will be closed when the last pending response has been received.
 
+Connection will also be closed if response entity has been cancelled (e.g. by attaching it to ``Sink.cancelled``)
+or consumed only partially (e.g. by calling ``take()`` on it). In order to prevent this behaviour entity should be
+explicitly drained by attaching it to ``Sink.ignore``.
```

Comment:
Maybe sprinkle some 'a's or 'an's to prefix nouns like 'connection' and 'entity' (2x)? Or just 'Connections'?